wifiphisher/common/interfaces.py

```diff
@@ -169,6 +170,52 @@ def __init__(self):
                "This is due to the specified mac address may be invalid")
         Exception.__init__(self, message)
 
+class DeauthInterfaceManagedByNMError(Exception):
+    """
+    Exception class to raise in case of NetworkManager controls the
+    deauth interface.
+    :param self: A DeauthInterfaceManagedByNMError object
+    :type self: DeauthInterfaceManagedByNMError
+    :return: None
+    :rtype: None
+    """
+    def __init__(self):
+        """
+        Construct the class.
+
+        :param self: An DeauthInterfaceManagedByNMError object
+        :type self: DeauthInterfaceManagedByNMError object
+        :return: None
+        :rtype: None
+        """
+
+        message = ("We have failed to specify the jamming interface. This due to "
+               "the specified interface is controlling by NetworkManager.")
+        Exception.__init__(self, message)
+
+class ApInterfaceManagedByNMError(Exception):
+    """
+    Exception class to raise in case of NetworkManager controls the
+    AP interface.
+    :param self: An ApInterfaceManagedByNMError object
+    :type self: ApInterfaceManagedByNMError
+    :return: None
+    :rtype: None
+    """
+    def __init__(self):
+        """
+        Construct the class.
+
+        :param self: An ApInterfaceManagedByNMError object
+        :type self: ApInterfaceManagedByNMError object
+        :return: None
+        :rtype: None
+        """
+
+        message = ("We have failed to specify the AP interface. This due to "
+               "the specified interface is controlling by NetworkManager.")
```

Comment:
We should also give instructions here. Otherwise we'll have plenty of open issues by users that need guidance.  

Something like:

"Interface wlan0 is controlled by NetworkManager. You need to manually set the devices that should be ignored by NetworkManager using the keyfile plugin (unmanaged-devices directive). For example, '[keyfile] unmanaged-devices=interface-name:wlan0' needs to be added in your NetworkManager configuration file."